src/watertap_contrib/reflo/costing/watertap_reflo_costing_package.py

```diff
@@ -377,6 +379,8 @@ class REFLOSystemCostingData(WaterTAPCostingBlockData):
     def build_global_params(self):
         super().build_global_params()
 
+        pyo.units.load_definitions_from_strings(["USD_2023 = 500/797.9 * USD_CE500"])
+
         self.base_currency = pyo.units.USD_2021
```

Comment:
Shou
```suggestion
        self.base_currency = pyo.units.USD_2023
```
Are you leaving this in purposely to avoid breaking tests with a new default?

Reply:
Yes exactly. Setting 2023 as the default would cause many unit model costing tests to fail and I just don't want to deal with that at the moment.

Reply:
I suppose an alternative would be to put `m.fs.costing.base_currency = pyunits.USD_2021` in all those tests

Reply:
That would be an alternative. Then you could add a test simply to verify the default currency for all 3 Costing blocks

Reply:
see the `_check_commpon_param_equivalence` method on `REFLOSystemCosting`

Reply:
see the `_check_common_param_equivalence` method for `REFLOSystemCosting`